docker_test.go

```diff
@@ -137,7 +138,7 @@ func TestDockerFatManifestPull(t *testing.T) {
 	buildNumber := "1"
 
 	// Pull docker image using docker client
-	artifactoryCli.Exec("docker-pull", imageTag, *tests.DockerVirtualRepo, "--build-name="+tests.DockerBuildName, "--build-number="+buildNumber)
+	artifactoryCli.Exec("docker-pull", imageTag, *tests.DockerRemoteRepo, "--build-name="+tests.DockerBuildName, "--build-number="+buildNumber)
```

Comment:
It looks like *DockerVirtualRepo* is no longer in use. Why don't we also test docker pull from a virtual repo, in addition to a remote repo?

Reply:
You're right, I will add both tests for virtual and remote!.